### ISSUE TITLE
mr_impersonate - accounts

### ISSUE THREAD

**Issue by jtagcat** [openmicroapps/unify#3]
pretty much a database

**Comment by prologic** [comment]
Why would we need this if a higher-level goal is to be self-hosted? We're not building a SaaS here right? :) -- Even though one option for monetizing the project would be to offer up SaaS hosted instances for a fee (_for the lazy or non-tech savvy_).

**Comment by jtagcat** [comment]
Oh pardon me, I had it made up in my head, add a 'login to save your data' or something, since cookies are kind of volatile.
We do not want to take users passwords, since it's a security risk and users don't need to log in often. E-mail verification login should do.

**Comment by prologic** [comment]
Yeah lets try to avoid this if we can. I agree we should support SaaS hosting for users that either can't or don't want to host themselves  and "impersonation" will be useful sometimes to debug user issues -- But OTOH it is a security hole and an invasion of privacy so I'm not that keen on it in the first place -- Built better tools, tracing and better ways to debug when things go wrong without prying on users :)

**Comment by jtagcat** [comment]
See https://github.com/openmicroapps/meta/issues/10
